source/manual/aws-iam-key-rotation.html.md

```diff
@@ -5,93 +5,131 @@ layout: manual_layout
-every 3 months.
+GOV.UK uses [AWS IAM] accounts to manage applications access to our AWS
+infrastructure. The [Access Keys] associated with these accounts need to be
+[rotated] every 3 months.
 
-## How to rotate Access Keys
+Rotating AWS IAM keys requires permissions which are available with the `admin`,
+`internal-admin`, `platformhealth-poweruser` or `poweruser` roles. If you can't
+assume one of these roles, you won't be able to rotate keys.
 
-> **Note**
->
-> Rotating AWS IAM keys requires permissions which are available with the
-> `admin`, `internal-admin`, `platformhealth-poweruser` or `poweruser` roles.
-> If you can't assume one of these roles, you won't be able to rotate keys.
+## How to rotate Access Keys
 
-Log in to the [IAM Management Console](https://console.aws.amazon.com/iam/home?region=eu-west-1#/users)
-for the environment you want to rotate keys for and you'll see a list of users
-and the age of their access keys.
+Log in to the [IAM Management Console] in the environment you want to rotate
+keys for and you'll see a list of users and the age of their access keys.
 
 ### 1. Find users access key
 
-Click on the user whose key you want to rotate. Click on `Security Credentials`. Make a note of the Access Key ID.
+- Click on the user whose key you want to rotate.
+- Click on the `Security Credentials` tab.
 
-### 2. Find where the key is used
+Users should always only have one access key. The only time a user should have
+two keys is when we are in the process of rotating the old one. For this reason,
+AWS allow users to have a maximum of two Access Keys at the same time.
 
-Enter [govuk-secrets/puppet](https://github.com/alphagov/govuk-secrets/tree/master/puppet) in your machine.
-Run one of the [Puppet AWS common actions](https://github.com/alphagov/govuk-secrets/tree/master/puppet_aws#common-actions) and search for the `Access Key ID` in the result.
+If this user has no keys, consider deleting it if it's not needed.
 
-> **Note**
-> The command you should run is related to the environment you need to target.
-> For example, to edit global staging credentials in the `govuk::apps` namespace, you need to run `bundle exec rake 'eyaml:edit[staging,apps]'`.
+If this user has only one key, make a note of the `Access Key ID` and proceed to
+[find out if the key is used].
+
+If this user has two keys, you'll need to follow this guide twice. The first
+time, you'll need to consider one key as the old one to rotate and the other key
+as if it was a new one just created. The second time, you'll follow the guide as
+normal because the user will then have only one key. Now, choose which key is
+the one you are going to rotate and apply the following steps to that key.
+
+### 2. Find out if the key is used
+
+- Check the `Last used` value.
+- If the value is a date, [find where the key is used].
```

Comment:
What if it's a really old date? Could it be that the key is no longer in use?

Reply:
It could be. It would mean that we've deleted the key but not removed it from AWS. 🤷‍♂
I'm not sure where to draw the line here though (a month? a year?) as some of our keys are apparently very rarely used.

Reply:
I think the line for "how old a key is that it isn't in use" depends on how often the key should be used, so maybe say something about that?

Reply:
That makes sense but I'm not sure how to find out how often a key should be used. Do we need to first find the key and - based on what it's used for - understand how often it should be used? 🤔

Reply:
I guess it's hopefully apparent once you understand how the key is used. I'm not sure it's possible (or useful) to be more specific than that.